e2etest/utils.go

```diff
@@ -50,11 +50,21 @@ func wgetRetry(address string, exc executor.Executor) error {
 	return err
 }
 
-func tweakServicePort(svc *v1.Service) {
+func tweakService(svc *v1.Service, ipFamily string) {
 	if servicePodPort != 80 {
 		// if servicePodPort is non default, then change service spec.
 		svc.Spec.Ports[0].TargetPort = intstr.FromInt(int(servicePodPort))
 	}
+	if ipFamily == "ipv6" {
```

Comment:
I am not sure to always do this. I'd prefer having only specified in the dual stack case, and keeping the default in the v4 / v6 case.